docker/mariner/Dockerfile.temurin-8-jdk

```
@@ -0,0 +1,28 @@
+ARG IMAGE=cblmariner.azurecr.io/base/core
+ARG TAG=2.0
```

To secure the supply chain, we need to remove the image.

```suggestion
ARG IMAGE
ARG TAG
```